src/loaders/svg/tvgSvgLoaderCommon.h

```diff
@@ -308,6 +314,7 @@ struct SvgLoaderData
     Array<SvgStyleGradient*> gradients;
     SvgStyleGradient* latestGradient = nullptr; //For stops
     SvgParser* svgParse = nullptr;
+    Array<SvgNodeIdPair*> cloneNodes;
```

Comment:
It looks too fragmented memory.
How about  Array<SvgNodeIdPair>?

Reply:
Could you describe more?

Reply:
@mmaciola sorry for this, \< , \> these tags are ignored by this github viewer. see again the above.

Reply:
ok. done